create-keystone-app/starter/package.json

```diff
@@ -8,22 +8,12 @@
     "build": "keystone build",
     "postinstall": "keystone postinstall"
   },
+  "// @aws-sdk": "temporary dependency until https://github.com/keystonejs/keystone/issues/8023 is resolved",
```

Comment:
Unfortunate, but we can remove it when https://github.com/keystonejs/keystone/issues/8023 is resolved